src/libraries/Microsoft.Extensions.Options/gen/Emitter.cs

```diff
@@ -20,11 +21,11 @@ internal sealed class Emitter : EmitterBase
         private const string StaticValidationResultType = "global::System.ComponentModel.DataAnnotations.ValidationResult";
         private const string StaticValidationAttributeType = "global::System.ComponentModel.DataAnnotations.ValidationAttribute";
 
-        private string _staticValidationAttributeHolderClassName = "__Attributes";
-        private string _staticValidatorHolderClassName = "__Validators";
-        private string _staticValidationAttributeHolderClassFQN;
-        private string _staticValidatorHolderClassFQN;
-        private string _modifier;
+        private readonly string _staticValidationAttributeHolderClassName = "__Attributes";
```

Comment:
Why aren't these `const string`?

Reply:
Because it can be concatenated with the generated suffix. I don't think the compiler will allow that.